chrome/content/zotero/zoteroPane.js

```diff
@@ -2786,6 +2786,16 @@ var ZoteroPane = new function()
 				return;
 			}
 			
+			//Special treatment for zotero links
+			if (uri.match(/^zotero\:\/\/(open-pdf|select)/)) {
+				let ioservice = Components.classes["@mozilla.org/network/io-service;1"]
+					  .getService(Components.interfaces.nsIIOService);
+				let zoteroLink = ioservice.newURI(uri, null, null);
+				openZoteroLink = Components.classes["@mozilla.org/network/protocol;1?name=zotero"]
+						.getService(Components.interfaces.nsIProtocolHandler).newChannel(zoteroLink);
+				return;
```

Comment:
(Protocol handler stuff is being totally overhauled on the api_syncing branch.)

`open-pdf` is a ZotFile thing. I'm not crazy about its being here, but it doesn't really matter.